src/submitButton/formSubmitButton.ts

```diff
@@ -22,6 +23,16 @@ submitButton.addEventListener('click', (event: Event) => {
     if (nameInput.value === "") displayNameRequiredError()
     if (projectInput.value === "") displayProjectRequiredError()
 
+    if (
+        !checkboxes[0].checked &&
+        !checkboxes[1].checked &&
+        !checkboxes[2].checked &&
+        !checkboxes[3].checked &&
+        !checkboxes[4].checked
+    ) {
+        displaySocialCheckboxError()
+    }
```

Comment:
I'm not entirely aware of the context, `checkboxes` might have more elements... but this might be helpful

```suggestion
    if (checkboxes.every(ch => !ch.checked)) {
        displaySocialCheckboxError()
    }
```

Reply:
I think you could even apply [De Morgan's law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) haha

```suggestion
    if (checkboxes.some(ch => ch.checked)) {
        displaySocialCheckboxError()
    }
```

Reply:
That's way more readable! Applied!